test/e2e/framework.go

```diff
@@ -126,12 +129,16 @@ const (
 
 	nginxLBService = "nginx-loadbalancer"
 
-	exporterActiveFlowExportTimeout     = 2 * time.Second
-	exporterIdleFlowExportTimeout       = 1 * time.Second
-	aggregatorActiveFlowRecordTimeout   = 3500 * time.Millisecond
-	aggregatorInactiveFlowRecordTimeout = 6 * time.Second
+	exporterActiveFlowExportTimeout        = 2 * time.Second
+	exporterIdleFlowExportTimeout          = 1 * time.Second
+	aggregatorActiveFlowRecordTimeout      = 3500 * time.Millisecond
+	aggregatorInactiveFlowRecordTimeout    = 6 * time.Second
+	aggregatorClickHouseBatchExportTimeout = 8 * time.Second
```

Comment:
May I ask why we set the timeout to 8 seconds here, and when the flow record should theoretically arrive on Clickhouse after we start the traffic?

Reply:
was considering the queueing for batch commit could introduce additional timeout before the record hits CH DB. Removed in the latest change.